solr/htdocs/solr/template_data.js

```diff
@@ -2007,7 +2002,7 @@
       }
     },
     noresultsnarrow: {
-      template: "<div class=\"scnarrow\">\n  <h1>No results for <em>thing</em> '<i class='search'>search</i>'</h1>\n  <ul>\n    <li class=\"wide\"><div>\n      You were searching the whole site, but still nothing was found.\n    </div></li>\n    <li class=\"narrow_rsid\"><div><strong>\n      You appear to have been searching for a variation rsid.\n      There may be new variants which have not yet been incorporated\n      into Ensembl. If this is the case, you may find information\n      about this variant on the\n      <a href=\"http://www.ncbi.nlm.nih.gov/snp\"\n      >NCBI website</a>\n    </strong></div></li>\n    <li class=\"narrow\"><div>\n      You were only searching <em>thing</em>.\n    </div></li>\n    <li class=\"narrow_any\"><div>\n      And there are <i class=\"count\">42</i> results in\n      <i class=\"all\">all</i> on the whole site.\n      <a href=\"#\">Search full site</a>.\n    </div></li>\n    <li class='narrow_none'><div>\n      But there are no results in\n      any category on the whole site, anyway.\n    </div></li>\n    <li><div class=\"roll_hidden\">\n      <a href=\"#\">More help on searching ...</a>\n      <div class=\"roll_hidden_text\">\n      </div>\n    </div></li>\n  </ul>\n</div>",
+      template: "<div class=\"scnarrow\">\n  <h1>No results for <em>thing</em> '<i class='search'>search</i>'</h1>\n  <ul>\n    <li class=\"wide\"><div>\n      You were searching the whole site, but still nothing was found.\n    </div></li>\n    <li class=\"narrow_rsid\"><div><strong>\n      You appear to have been searching for a variation rsid.\n      There may be new variants which have not yet been incorporated\n      into Ensembl. If this is the case, you may find information\n      about this variant on the\n      <a href=\"http://www.ncbi.nlm.nih.gov/sites/entrez\"\n      >NCBI website</a>\n    </strong></div></li>\n    <li class=\"narrow\"><div>\n      You were only searching <em>thing</em>.\n    </div></li>\n    <li class=\"narrow_any\"><div>\n      And there are <i class=\"count\">42</i> results in\n      <i class=\"all\">all</i> on the whole site.\n      <a href=\"#\">Search full site</a>.\n    </div></li>\n    <li class='narrow_none'><div>\n      But there are no results in\n      any category on the whole site, anyway.\n    </div></li>\n    <li><div class=\"roll_hidden\">\n      <a href=\"#\">More help on searching ...</a>\n      <div class=\"roll_hidden_text\">\n      </div>\n    </div></li>\n  </ul>\n</div>",
```

Comment:
Is changing the destination URL intended ?

Reply:
This is a change that has been automatically compiled from coffeescript.

Reply:
Which coffeescript file ? (It feels familiar but I can't find the relevant PR to check its not a reversion)

Reply:
This link was updated directly in the js file instead of in the coffeescript source 😱 

https://github.com/Ensembl/public-plugins/commit/1c524e2c3ab8d9d52abf0eb929ff4ea02fa1b663

Reply:
Ouch. Can you update the coffeescript to change the URL permanently ?

Reply:
> Which coffeescript file ?
solr/src/templates/rhs_templates.coffee

I updated the link there and recompiled the files.